src/components/CodeChangesTable/index.tsx

```diff
@@ -1,65 +1,69 @@
-import { Table } from "antd";
-import { ColumnProps } from "antd/es/table";
+import { useRef } from "react";
+import { useLeafyGreenTable } from "@leafygreen-ui/table";
 import { FileDiffText } from "components/CodeChangesBadge";
 import { StyledLink, WordBreak } from "components/styles";
+import { BaseTable } from "components/Table/BaseTable";
+import { TablePlaceholder } from "components/Table/TablePlaceholder";
 import { FileDiffsFragment } from "gql/generated/types";
 
 interface CodeChangesTableProps {
   fileDiffs: FileDiffsFragment[];
-  showHeader?: boolean;
 }
 export const CodeChangesTable: React.FC<CodeChangesTableProps> = ({
   fileDiffs,
-  showHeader = true,
-}) => (
-  <Table
-    data-cy="code-changes-table"
-    rowKey={rowKey}
-    columns={columns(showHeader)}
-    dataSource={fileDiffs}
-    pagination={false}
-    showHeader={showHeader}
-  />
-);
+}) => {
+  const tableContainerRef = useRef<HTMLDivElement>(null);
+  const table = useLeafyGreenTable({
+    columns,
+    data: fileDiffs ?? [],
+    containerRef: tableContainerRef,
+    enableColumnFilters: false,
+    enableSorting: false,
+  });
 
-const rowKey = (record: FileDiffsFragment): string =>
-  `${record.diffLink}_code_table`;
+  return (
+    <BaseTable
+      data-cy="code-changes-table"
+      data-cy-row="code-changes-table-row"
+      emptyComponent={<TablePlaceholder message="No diffs." />}
```

Comment:
[nit] I don't think you need this because we don't show the table if there are no diffs
<img width="918" alt="image" src="https://github.com/evergreen-ci/spruce/assets/9298431/c2d21467-3a4d-4314-ba1e-f00f29ac075b">